bin/server

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env sh
 
-bin/rails server --port 5000 --binding lvh.me
+set -e
+
+docker-compose up
```

Comment:
Should we use `-d` here too?

Reply:
Current behavior for `bin/server` is to run all required processes and display log messages right into the console.